GDPRuler.py

```diff
@@ -39,21 +40,23 @@ def main():
   # Open the controller process
   process_args = [os.path.join(os.path.dirname(os.path.abspath(__file__)), './controller/build/gdpr_controller')]
   process_args += ['--db', args.db]
-  if args.address:
-    process_args += ['--address', args.address]
+  if args.db_address:
+    process_args += ['--db_address', args.db_address]
   process_args += ['--logpath', args.logpath]
   if args.db_encryptionkey:
     process_args += ['--db_encryptionkey', args.db_encryptionkey]
   if args.log_encryptionkey:
     process_args += ['--log_encryptionkey', args.log_encryptionkey]
+  process_args += ['--frontend_address', args.frontend_address]
+  process_args += ['--frontend_port', args.frontend_port]
```

Comment:
minor suggestion: could `controller_address` and `controller_port` be a bit more intiutive?

Reply:
Done